web/modules/custom/hoeringsportal_dialogue/translations/hoeringsportal_dialogue.da.po

```diff
@@ -1,101 +1,67 @@
-# Danish translation of dialogue module
+# Danish translation of 
 #
 msgid ""
 msgstr ""
 "Project-Id-Version: PROJECT VERSION\n"
-"POT-Creation-Date: 2025-11-20 14:19+0100\n"
-"PO-Revision-Date: 2025-11-20 14:19+0100\n"
+"POT-Creation-Date: 2025-12-16 15:13+0100\n"
+"PO-Revision-Date: 2025-12-16 15:13+0100\n"
 "Last-Translator: NAME <EMAIL@ADDRESS>\n"
 "Language-Team: LANGUAGE <EMAIL@ADDRESS>\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"Language: da\n"
 
+#, fuzzy
+msgid "Comment @commentId and it's children have been unpublished."
+msgstr ""
+
+#, fuzzy
 msgid "Dialogue proposals"
-msgstr "Dialogforslag"
-msgid "Proposal replies"
-msgstr "Svar til dialogforslag"
-msgid "Proposals"
-msgstr "Forslag"
-msgid "Comments"
-msgstr "Kommentarer"
-msgid "like this"
-msgstr "synes om"
-msgid "Participate with your proposal"
-msgstr "Deltag med dit forslag"
+msgstr ""
+
+#, fuzzy
 msgid "Early inclusion"
-msgstr "Tidlig dialog"
```

Comment:
Can we keep these translations or should they be made again?

Reply:
Restored in https://github.com/itk-dev/deltag.aarhus.dk/pull/576/changes/25a1b18840da5c302fe892748e998ee20d1ef6fa.